shared/contexts/StatsProvider.tsx

```diff
@@ -54,6 +64,11 @@ export function StatsProvider (props: PropsWithChildren<any>): JSX.Element | nul
     }
   }, [api, interval, network, dispatch])
 
+  // Global polling based on blockCount and network, so no need to fetch per page
+  useEffect(() => {
```

Comment:
No comment on the functionality-wise, this branch is too fast that I thought i was using a samsung galaxy s32 ultra pro max 4k phone

But i have one comment on adding the pooling for `fetchPoolPairs` here because it doesn't contribute to the main purpose of `StatsProvider`, unless we change the file name to something more generic, would it work the same if you were to move this to maybe `<Main>`?

Reply:
Oh yeah good suggestion, let me move it. After this release, I'll move `fetchTokens` also. But it will affect lots of pages so not this release

Reply:
Updated